capellambse/svg/drawing.py

```diff
@@ -389,6 +387,12 @@ def add_port(
             elif class_ in decorations.component_ports:
                 port_id = "#ComponentPortSymbol"
 
+            if port_id not in self.deco_cache:
+                self.__drawing.defs.add(
+                    decorations.deco_factories[port_id[1:]]()
+                )
+                self.deco_cache.append(port_id)
+
             grp.add(
                 self.__drawing.use(
                     href=port_id,
```

Comment:
IMO it makes more sense to add the `#` sign only where it's actually needed, instead of stripping it out where it would hurt. This way, there's this leading `#` in the `deco_cache` only for ports, but not other symbols, which seems somewhat odd to me.